contents/docs/api/trends.mdx

```diff
@@ -1,80 +1,320 @@
+| `properties` | Array of Properties | The key/values that you want to filter on. Basic usage is: `[{"key": "$browser", "value": "Chrome"}]`<br /><br />For each property, you can specify:<br />- `key` Key of the property<br />- `value` Value you want to filter on<br />- `type` Either `person` or `event`<br />- `operator`, any of: <br />-- `exact`/empty<br />--`is_not`<br />--`icontains`<br />--`not_icontains`<br />--`gt`<br />--`lt`<br />--`is_set` | ❌ |
+| `interval` | String | Interval of the data. `minute`, `hour`, `day`, `week`, `month` | ❌ |
+| `breakdown_value` | String | Split out all entities by any property value or cohort id. | ❌ |
+| `breakdown_type` | String | Default: `property`. Can be set to `cohort`, in which case you can pass through a cohort_id to `breakdown_value` | ❌ |
+| `from_dashboard` | String | | ❌ |
+| `shown_as` | String / `Volume`, and `Stickiness` |  | ❌ |
+
+
+
+</HiddenSection>
+
+<HiddenSection headingType='h4' title='Example'>
+
+##### Request
+
+```
+GET /api/insight/trend/?events=[{"id":"$pageview"}]&interval=week
+```
+
+##### Response
+
+```json
+HTTP 200 OK
+Content-Type: application/json
+
+{
+    "result": [
+        {
+            "action": {
+                "id": "$pageview",
+                "type": "events",
+                "order": null,
+                "name": "$pageview",
+                "math": null,
+                "math_property": null,
+                "properties": []
+            },
+            "label": "$pageview",
+            "count": 84.0,
+            "data": [
+                0.0,
+                69.0,
+                15.0
+            ],
+            "labels": [
+                "Sun. 18 April",
+                "Sun. 25 April",
+                "Sun. 2 May"
+            ],
+            "days": [
+                "2021-04-18",
+                "2021-04-25",
+                "2021-05-02"
+            ]
+        }
+    ],
+    "next": null
+}
+```
+
+</HiddenSection>
+
+<Endpoint endpoint='/api/insight/session' allowedMethods={['get']} />
+
+Get a list of sessions from PostHog.
+
+<HiddenSection headingType='h4' title='Query Parameters'>
+
+
+| Parameter | Type / Valid Values | Description | Required |
+| --- | --- | --- | --- |
+| `properties` | Array of Properties | The key/values that you want to filter on. Basic usage is: `[{"key": "$browser", "value": "Chrome"}]`<br /><br />For each property, you can specify:<br />- `key` Key of the property<br />- `value` Value you want to filter on<br />- `type` Either `person` or `event`<br />- `operator`, any of: <br />-- `exact`/empty<br />--`is_not`<br />--`icontains`<br />--`not_icontains`<br />--`gt`<br />--`lt`<br />--`is_set` | ❌ |
+| `session` | String |Specifies the type of a session. For example, `avg`, or `dist`. | ❌ |
```

Comment:
This isn't exactly the "type" of a session, but rather the type of operation to perform on session duration values